### ISSUE TITLE
VSCode v1.29.1: File icons in IntelliSense not showing

### ISSUE THREAD

**Issue by ThePiranha** [microsoft/vscode#64843]
Hi VSCode Team,

**Problem#01:**

After upgrading to v1.29.1 files vscode icon which is associated with VSCode is not showing up in any icon view.

The default program in following file is set as VSCode and before upgrading to v1.29.1 vscode icon was showing there.
![associatedfile](https://user-images.githubusercontent.com/9007981/49849511-20e4f400-fe04-11e8-8e6e-e8258db5ca4c.png)

Default app setup:
![associatedprogram](https://user-images.githubusercontent.com/9007981/49849519-280c0200-fe04-11e8-8a6b-f9be5c4e793b.png)

**Problem#02:**

In the latest October 2018 release v1.29.1 a new feature added **[Files and folder icons in IntelliSense](https://code.visualstudio.com/updates/v1_29#_editor)** where in IntelliSense it show the icons associated with that file type. But when I try to import a css file the icon is not showing there and even the files that is resides in the css file folder not showing up in intellisense suggestion.

![noshowing](https://user-images.githubusercontent.com/9007981/49849804-6524c400-fe05-11e8-9724-c342c47bdb6e.png)

In the above in the folder where the foo.css resides there is other .css, .ts files but it is not showing when I called the "@import" command. Or, is there something I am missing?

For both above issue my vscode version is:

![vscodev](https://user-images.githubusercontent.com/9007981/49850149-b41f2900-fe06-11e8-8764-5cf408a224bf.png)

You can refer to my stream to reproduce the issue where I try to use the Files and folder icons in IntelliSense: https://www.twitch.tv/videos/347992950

**Comment by misolori** [comment]
For problem 2, I'd make sure you open the project folder instead of dragging in files (from your stream it appears you are dragging in files, which has no context). I can confirm that this works on 1.29.1

<img width="830" alt="image" src="https://user-images.githubusercontent.com/35271042/49915886-e2bff280-fe4c-11e8-918f-254600de58ad.png">

**Comment by ThePiranha** [comment]
@mjbvz and @misolori 
![image](https://user-images.githubusercontent.com/9007981/50050161-c3131d80-011d-11e9-9ec3-e61f9233d031.png)
Looks like my problem 2 is solved now. File icon is showing in v1.29.1 by opening the project folder.

![working](https://user-images.githubusercontent.com/9007981/50050194-6ebc6d80-011e-11e9-87af-9bd9d6b6c10b.png)


And for problem 1 I will later try to re-install vscode. In the meantime let me try if I can figure out a way which does not require re-installing vscode.

**Comment by kotuck** [comment]
Problem#01
Hello, I managed to get some data that may be useful. I will be happy to provide any other information or screenshots. I have one version of vs code installed, I've tried to reinstall it as you recommended as well.

I deleted associations for .json and .js  files:
\HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\FileExts\*
\HKEY_CLASSES_ROOT
The first time I just opened the file and select vs code, the second time I opened it as Open with > Choose another app.

Not correct
\HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\FileExts\.json\UserChoice
ProgId=Applications\Code.exe (after the first selection)
ProgId=json_auto_file (after re-selection)
It has a gray and blue icon in the selection menu

Correct:
\HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\FileExts\.js\UserChoice
ProgId=VSCode.js (works well after the first selection)
It has a black and blue icon in the selection menu

Version: 1.30.2 (user setup)
Commit: 61122f88f0bf01e2ac16bdb9e1bc4571755f5bd8
Date: 2019-01-07T22:54:13.295Z
Electron: 2.0.12
Chrome: 61.0.3163.100
Node.js: 8.9.3
V8: 6.1.534.41
OS: Windows_NT x64 10.0.17763
